pmda/parallel.py

```diff
@@ -284,6 +281,69 @@ def _single_frame(self, ts, atomgroups):
         """
         raise NotImplementedError
 
+    def prepare_jobs(self,
```

Comment:
*prepare_jobs* sounds confusing to me – what "jobs"? If it's part of the documented workflow then it could just be *prepare*. 

*prepare_dask* would be more explicit but also a bit pointless because PMDA is fully intertwined with dask so that's the only thing we would be preparing for. *create_dask_graph* is too long and really talks to much about implementation details.

All in all, I'd just call it *prepare* and add more docs stating clearly what is being prepared and under which circumstances a user needs to run it.